ui/pages/confirmations/components/confirm/ledger-info/ledger-info.tsx

```diff
@@ -36,7 +36,11 @@ const LedgerInfo: React.FC = () => {
   const t = useI18nContext();
   const dispatch = useDispatch();
 
-  const webHidConnectedStatus = useSelector(getLedgerWebHidConnectedStatus);
+  const inE2eTest =
+    process.env.IN_TEST && process.env.JEST_WORKER_ID === 'undefined';
```

Comment:
:information_source:  we need to be specific, as only using IN_TEST will affect our unit tests, but that's undesired, as in unit tests we have scenarios to cover the connect errors etc, which can only be triggered if we are not connected